recipe/meta.yaml

```
@@ -1,39 +1,83 @@
 {% set name = "evidently" %}
-{% set version = "0.3.3" %}
+{% set version = "0.4.39" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/evidentlyai/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 49f3f5a724e65369f33243732dacd48aad68e3aa0bd66e3a8e7784ed5f675b5f
+  sha256: c07a65a5c07138708328dbdf1e2b369f725039f714efb393b5631db7b57fa393
+  patches:
+    - 0001-patch-test-to-exclude-experimental-features.patch
 
 build:
   number: 0
   # umap-learn isn't available on s390x
   skip: True  # [py<37 or s390x]
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  # missing dynaconf for py312
+  skip: True  # [py>311]
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+    # removing experimental features that are dependent on missing chromadb
+    - rm -rf $SP_DIR/evidently/experimental
 
```

```suggestion
  entry_points:
    - evidently=evidently.cli:app

```
https://github.com/evidentlyai/evidently/blob/v0.4.39/setup.py#L115C40-L115C67

---

good catch!